code/style.css

```diff
@@ -1,58 +1,145 @@
-body {
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body,
+html {
+  height: 100%;
   margin: 0;
   font-family: "Courier New", Courier, monospace;
 }
 
+main {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: column;
+  justify-content: center;
+  background-color: #07080b;
+}
+
 p {
   margin: 1px;
   font-weight: 300;
-  background-color: #fff;
+  background-color: transparent;
+}
+
+.hero-image {
+  height: 50%;
+  background-position: center;
+  background-repeat: no-repeat;
+  display: flex;
+  flex-wrap: wrap;
+  object-fit: cover;
+}
+
+.hero-text {
+  text-align: center;
+  position: absolute;
+  top: 5%;
+  left: 5%;
+  color: rgb(226, 223, 214);
+  text-shadow: 0 0 20px #fefcc9, 10px -10px 30px #feec85,
+    -20px -20px 40px #ffae34, 20px -40px 50px #ec760c, -20px -60px 60px #cd4606,
+    0 -80px 70px #973716, 10px -90px 80px #451b0e;
 }
 
 .cards {
-  margin: 50px auto;
+  margin: auto;
+  margin-top: -150px;
   width: 90%;
-  height: 600px;
-  background: #ccc;
+  height: auto;
   display: flex;
+  flex-wrap: wrap;
   justify-content: space-around;
   align-items: center;
+  color: black;
+  text-shadow: 2px 2px 0 rgba(255, 0, 180, 0.5);
 }
 
 .killer-deck {
-  width: 250px;
-  height: 400px;
+  width: 300px;
+  height: 500px;
   background: #000;
   text-align: center;
+  margin-bottom: 10px;
   padding-top: 30px;
   border-radius: 16px;
+  -moz-box-shadow: inset 3px 3px 5px 6px #ccc;
+  -webkit-box-shadow: inset 3px 3px 5px 6px #ccc;
+  box-shadow: inset 3px 3px 5px 6px #ccc;
 }
 
 .weapon-deck {
-  width: 250px;
-  height: 400px;
-  background: rgb(10, 53, 10);
+  width: 300px;
+  height: 500px;
+  background: rgb(143, 141, 116);
   text-align: center;
+  margin-bottom: 10px;
   padding-top: 30px;
   border-radius: 16px;
+  -moz-box-shadow: inset 3px 3px 5px 6px #ccc;
+  -webkit-box-shadow: inset 3px 3px 5px 6px #ccc;
+  box-shadow: inset 3px 3px 5px 6px #ccc;
 }
 
 .room-deck {
-  width: 250px;
-  height: 400px;
-  background: midnightblue;
-
+  width: 300px;
+  height: 500px;
+  background: #bdbdb7;
   text-align: center;
+  margin-bottom: 10px;
   padding-top: 30px;
   border-radius: 16px;
+  -moz-box-shadow: inset 3px 3px 5px 6px #ccc;
+  -webkit-box-shadow: inset 3px 3px 5px 6px #ccc;
+  box-shadow: inset 3px 3px 5px 6px #ccc;
 }
 
 button {
   padding: 8px 16px;
-  background: crimson;
+  background: magenta;
   border: none;
-  font-size: 20px;
-  border-radius: 8px;
+  font-size: 15px;
+  border-radius: 30px;
   display: block;
   margin: 0 auto;
 }
+
+footer {
+  display: flex;
+  flex-direction: column;
+  position: relative;
+  margin-top: 40px;
+  margin-bottom: 5px;
+}
+
+footer h2 {
+  text-align: center;
+  margin-top: 5px;
+  margin-bottom: 20px;
+  color: rgb(226, 223, 214);
+}
+
+/* MEDIA QUERIES */
+
+@media screen and (min-width: 768px) {
+  .hero-text {
+    font-size: 20px;
+  }
+}
```

Comment:
Great job making your page responsive! Double check the borders for the mobile and tablet (I assume the ones with font-size 20px and 25 px, respectively), you're almost there! :)